### PR TITLE
Project-labyrinth - Therese Lindén & Sara Heijkenskjöld

### PR DESCRIPTION
**Pull request by thereselinden**
**Link to my deployed project:**
https://getlostwiththereseandsara.netlify.app/

**My reflections on how this week's project turned out:**
We are happy with the project, this week we tried to implement a CSS framework. It did not work out as we wanted on Safari and FireFox. If we had more time we would have investigate how to solve this styling problem. 

**Things I'd like to have clarified or explained in more detail:**

### CODE REVIEW

**Review by karolinalbinsson — APPROVED** [review]
Hello Sara and Therese! 
Nice work with this weeks project. I think it looks really good. Your design is awesome and made me feel very nostalgic! 

I can tell you have put a lot of attention into the details and look&feel of the game.
It is also clear that you have a good understanding of redux. 

I have made some comments that you can have a look at, but nothing that keeps this project from getting approved. goooood job!

 ⭐ ⭐ ⭐

File: code/src/reducers/ui.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export const ui = createSlice({
+  name: 'ui',
+  initialState: {
+    isLoading: false,
+  },
+
+  reducers: {
+    setLoading: (state, action) => {
```
Comment (HIPPIEKICK) [inline code/src/reducers/ui.js:10]: Nice!

File: code/src/reducers/game.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+    setDirection: (state, action) => {
+      if (state.game.coordinates) {
+        state.history = [...state.history, state.game];
+        state.previousSteps = [...state.previousSteps, state.game.coordinates];
+      }
+      state.game = action.payload;
+    },
+
+    historyGoBack: state => {
```
Comment (HIPPIEKICK) [inline code/src/reducers/game.js:34]: To be consistent with the naming, this could be called just `goBack`

File: code/src/components/GameMap.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+import { useSelector } from 'react-redux';
+
+import MapCoordinates from './MapCoordinates';
+
+const GameMap = () => {
```
Comment (HIPPIEKICK) [inline code/src/components/GameMap.js:7]: Super nice visualisation 💯

**Pull request closed without merging**